docs-chef-io/content/inspec/resources/registry_key.md

```diff
@@ -153,6 +163,12 @@ The `name` matcher tests the value for the specified registry setting:
 
     its('name') { should eq 'value' }
 
+### be_inherit
+
+The `be_inherit` matcher returns the `Boolean`. It will return `true` if registry key has inheritance enabled.
+
+    it { should be_inherit }
```

Comment:
For proper English, I thing this should be "be_inherited".

Reply:
Thanks, @clintoncwolfe. I have updated it.